src/js/components/Section.js

```diff
@@ -2,12 +2,16 @@
 
 var React = require('react');
 var Box = require('./Box');
+var SkipLinkAnchor = require('./SkipLinkAnchor');
+var merge = require('lodash/object/merge');
 
 var CLASS_ROOT = "section";
 
 var Section = React.createClass({
 
-  propTypes: Box.propTypes,
+  propTypes: merge(Box.propTypes, {
+    primary: React.PropTypes.bool
```

Comment:
Our Docs project is an example. On the main page we want the Section to be the main content, not the article, so we can skip the Design and Develop links on the top right corner. The same for the the Design and Develop pages.